src/sentry/static/sentry/app/views/issueList/header.tsx

```diff
@@ -35,8 +35,10 @@ type WrapGuideProps = {
 function WrapGuideTabs({children, tabQuery, query, to}: WrapGuideProps) {
   if (isForReviewQuery(tabQuery)) {
     return (
-      <GuideAnchor target="inbox_guide_tab" disabled={isForReviewQuery(query)} to={to}>
-        <GuideAnchor target="for_review_guide_tab">{children}</GuideAnchor>
+      <GuideAnchor target="inbox_guide_tab" disabled={!isForReviewQuery(query)} to={to}>
```

Comment:
we wanted inbox_guide_tab to display when the current query is not the for review tab and then it would take you to the inbox tab